src/metadataGeneration/methodGenerator.ts

```diff
@@ -57,7 +56,7 @@ export class MethodGenerator {
   }
 
   private buildParameters() {
-    const parameters = this.node.parameters.map((p) => {
+    const map = this.node.parameters.map((p) => {
```

Comment:
It might be nicer to just add the `filter` to the end of the `map`, because the intermediate result isn't really a `map`.

Reply:
done